### PR TITLE
Update buffer map initialization

### PR DESCRIPTION
**Pull request by ehanley324**
Now that we have a global entry names registry, the map of entrytype to buffer should be initialized with values from that table.

***
<a href="https://pr-preview.s3.amazonaws.com/ehanley324/performance-timeline/pull/135.html" title="Last updated on Jun 25, 2019, 6:14 PM UTC (d2094cf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/performance-timeline/135/def3145...ehanley324:d2094cf.html" title="Last updated on Jun 25, 2019, 6:14 PM UTC (d2094cf)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -292,8 +296,9 @@ <h2>The <dfn>PerformanceEntry</dfn> interface</h2>
         <dd>
           This attribute MUST return the type of the interface represented by
           this <a>PerformanceEntry</a> object.
-          <p class="note">Example `entryType` values defined by other
-          specifications include: <code>"mark"</code> and
+          <p class="note">All `entryType` values are defined in the <a href=
```
Comment (yoavweiss) [inline index.html]: Nit: I'd add a "relevant" after "the", so that it'd be "the relevant registry"

**Comment by ehanley324** [comment]
> Can we also change the part where we say the map is 'initially empty'? We already know how it should be initialized.

Done.

File: index.html (hunk 1 above)
Comment (ehanley324) [reply index.html]: Done.

**Comment by ehanley324** [comment]
> lgtm. looks like this has a merge conflict

Should be resolved now.

**Pull request merged**